docusaurus/docs/Android/03-ui/04-components/01-channel-list.md

```diff
@@ -29,27 +69,46 @@ channelListView.setChannelInfoClickListener { channel ->
 channelListView.setUserClickListener { user ->
     // Handle Member Click
 }
-```
+``` 
 
 The full list of available listeners is available [here](https://getstream.github.io/stream-chat-android/stream-chat-android-ui-components/stream-chat-android-ui-components/io.getstream.chat.android.ui.channel.list/-channel-list-view/index.html).
 
 ## Customization
 
-### Customization with XML Attributes
+There are two ways to customize the appearance of `ChannelListView`: 
+
+* Using XML attributes
+* Using the `TransformStyle` API at runtime to call `ChannelListView` setter functions
+
+### Customization with XML attributes
+There are multiple XML attributes that can be used to customize the appearance of the channel list. The most useful ones include:
+
+* `app:streamUiChannelDeleteEnabled` Specifying if the delete icon should be displayed.
+* `app:streamUiChannelDeleteIcon` A drawable reference for channel delete icon.
+* `app:streamUiChannelTitleTextColor` Color of channel title text.
+* `app:streamUiChannelTitleTextSize` Size of channel title text.
+* `app:streamUiChannelTitleFontAssets` Font assets reference for title font customization.
+* `app:streamUiChannelTitleTextFont` A font name for channel title.
+* `app:streamUiLastMessageTextSize` Size of last message text.
+* `app:streamUiLastMessageTextColor` Color of last message text.
+* `app:streamUiForegroundLayoutColor` A foreground color of channel list item.
+* `app:streamUiBackgroundLayoutColor` A background color of channel list item, visible when swiping the list item.
 
-<!-- TODO: Customization description -->
+
+The full list of available XML attributes is available under `ChannelListView` styleable [resource](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/attrs_channel_list_view.xml).
 
 ### Customization at Runtime
 
 <!-- TODO: Review this example, possibly remove it. -->
 
-Let's make an example and modify _ChannelListViewStyle_ programmatically. We want to change the title text appearance, some default icons, and colors, and disable some options:
+The following example shows how to use `ChannelListView` setter functions to:
```

Comment:
It would probably be good to map the order of these changes to the order they are changed in the code. This would create paralellism between the natural language instructions and the code example.